Error in production build

<!--
We will close this issue if you don't provide the needed information.

Please remember, the github issues is __NOT__ for support requests and general questions. It is for bugs and feature requests only.
Please read https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.
-->

### Issue type

**I'm submitting a ...**  (check one with "x")

* [ x] bug report
* [ ] feature request

### Issue description
I'm using v2.0.0-rc.10. The production build is producing the following error: 
**Current behavior:**
<!-- Describe how the bug manifests. -->
```
[ERROR] Error: Parse error on line 1:                                                                                                                                   
[ERROR] (* 1.25) + 4px  
```   
**Expected behavior:**
<!-- Describe what the behavior would be without the bug. -->

**Steps to reproduce:**
<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application. -->

**Related code:**
<!-- 
If you are able to illustrate the bug or feature request with an example, please provide a sample application via one of the following means:

A sample application via GitHub

StackBlitz (https://stackblitz.com)

Plunker (http://plnkr.co/edit/cpeRJs?p=preview)

-->

```
insert short code snippets here
```

### Other information:

**npm, node, OS, Browser**
```
npm: v6.4.1
node: v8.12.0
OS: Windows 10
Browser: Chrome
```

**Angular, Nebular**
```
Angular : 6.1.0
Nebular: v2.0.0-rc.10
```

Steps to reproduce:
 I'm using Nebular with JHipster v5.3.4.
1. Create a jhipster project
2. Integrate Nebular.
3. Build for production

Following the complete error log:

```
[ERROR] Error: Parse error on line 1:
[ERROR] (* 1.25) + 4px
[ERROR] Expecting 'SUB', 'LPAREN', 'NESTED_CALC', 'NUMBER', 'CSS_VAR', 
'LENGTH', 'ANGLE', 'TIME', 'FREQ', 'RES', 'EMS', 'EXS', 'CHS', 'REMS', 
'VHS', 'VWS', 'VMINS', 'VMAXS', 'PERCENTAGE', got 'MUL'
[ERROR]     at Parser.parseError (D:\Projects\arif\fb- 
quiz\node_modules\cssnano-preset-default\node_modules\reduce-css- 
calc\dist\parser.js:160:21)
[ERROR]     at Parser.parse (D:\Projects\arif\fb-quiz\node_modules\cssnano- 
preset-default\node_modules\reduce-css-calc\dist\parser.js:226:22)
[ERROR]     at D:\Projects\arif\fb-quiz\node_modules\cssnano-preset- 
default\node_modules\reduce-css-calc\dist\index.js:35:30
[ERROR]     at walk (D:\Projects\arif\fb-quiz\node_modules\postcss-value- 
parser\lib\walk.js:15:13)
[ERROR]     at ValueParser.walk (D:\Projects\arif\fb- 
quiz\node_modules\postcss-value-parser\lib\index.js:18:5)
[ERROR]     at exports.default (D:\Projects\arif\fb- 
quiz\node_modules\cssnano-preset-default\node_modules\reduce-css- 
calc\dist\index.js:29:51)
[ERROR]     at transformValue (D:\Projects\arif\fb- 
quiz\node_modules\cssnano-preset-default\node_modules\postcss- 
calc\dist\lib\transform.js:24:45)
[ERROR]     at exports.default (D:\Projects\arif\fb- 
quiz\node_modules\cssnano-preset-default\node_modules\postcss-
calc\dist\lib\transform.js:54:100)
[ERROR]     at D:\Projects\arif\fb-quiz\node_modules\cssnano-preset-default\node_modules\postcss-calc\dist\index.js:28:52
[ERROR]     at D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:144:26
[ERROR]     at Rule.each (D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:110:22)
[ERROR]     at Rule.walk (D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:143:21)
[ERROR]     at D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:146:32
[ERROR]     at Root.each (D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:110:22)
[ERROR]     at Root.walk (D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\postcss\lib\container.js:143:21)
[ERROR]     at D:\Projects\arif\fb-quiz\node_modules\cssnano-preset-default\node_modules\postcss-calc\dist\index.js:25:9
[ERROR]     at initializePlugin (D:\Projects\arif\fb-quiz\node_modules\optimize-css-assets-webpack-plugin\node_modules\cssnano\dist\index.js:35:51)
```

--- a/src/framework/bootstrap/styles/_hero-buttons.scss
+++ b/src/framework/bootstrap/styles/_hero-buttons.scss
@@ -604,8 +604,8 @@
   @return btn-hero-line-height(nb-theme(btn-font-size-sm));
 }
 
-@function btn-hero-line-height-tn() {
-  @return btn-hero-line-height(nb-theme(btn-font-size-tn));
+@function btn-hero-line-height-xs() {
+  @return btn-hero-line-height(nb-theme(btn-font-size-xs));
 }
 
 
@@ -626,7 +626,7 @@
   }
 
   &.btn.btn-tn {
-    line-height: btn-hero-line-height-tn();
+    line-height: btn-hero-line-height-xs();
   }
 }
 

--- a/src/framework/theme/components/button/_button-heroes.scss
+++ b/src/framework/theme/components/button/_button-heroes.scss
@@ -596,7 +596,7 @@
     line-height: btn-hero-line-height-sm();
   }
 
-  &.btn.btn-xs {
+  &.btn.btn-tn {
     line-height: btn-hero-line-height-xs();
   }
 }

--- a/src/framework/theme/components/button/_button-heroes.scss
+++ b/src/framework/theme/components/button/_button-heroes.scss
@@ -575,8 +575,8 @@
   @return btn-hero-line-height(nb-theme(btn-font-size-sm));
 }
 
-@function btn-hero-line-height-tn() {
-  @return btn-hero-line-height(nb-theme(btn-font-size-tn));
+@function btn-hero-line-height-xs() {
+  @return btn-hero-line-height(nb-theme(btn-font-size-xs));
 }
 
 
@@ -596,8 +596,8 @@
     line-height: btn-hero-line-height-sm();
   }
 
-  &.btn.btn-tn {
-    line-height: btn-hero-line-height-tn();
+  &.btn.btn-xs {
+    line-height: btn-hero-line-height-xs();
   }
 }
 

